balancer/weightedtarget/weightedtarget_test.go

```diff
@@ -1249,6 +1249,46 @@ func (s) TestInitialIdle(t *testing.T) {
 	}
 }
 
+// TestSubBalancerStateLazyUpdate covers the case that if the child reports a
```

Comment:
This is not a case of lazy update :)
We are ignoring these state transitions to reduce churn. If we did not do this, RPCs blocked on this sub-balancer being in `TF` will be retried when the sub-balancer moves to `Connecting`. But there is no good that will come out of it, since the sub-balancer is still in `Connecting` and the RPC will again be queued till there is another state transition. So, the name of this test should be something like `TestIgnoreSubBalancerStateTransitions`.

Reply:
SG. I was thinking of that the updates to the aggregate state was lazy :P